Test/ProGetUniversalPackage.Tests.ps1

```diff
@@ -903,7 +903,7 @@ Describe 'ProGetUniversalPackage.when path to third-party item does not exist' {
         $Global:Error.Clear()
 
         { Invoke-WhiskeyTask -TaskContext $context -Parameter @{ Name = 'fubar' ; Description = 'fubar'; Include = 'fubar'; Path = '.'; ThirdPartyPath = 'fubar' } -Name 'ProGetUniversalPackage' } | Should -Throw
-        $Global:Error | Should -BeLike ('* ThirdPartyPath`[0`] "{0}*" does not exist.' -f (Join-Path -Path $context.BuildRoot -ChildPath 'fubar'))
+        $Global:Error | Select-Object -First 1 | Should -Match '\bThirdPartyPath\b.*\bfubar\b.*\ does\ not\ exist' 

```

Comment:
Spaces shouldn't need to be escaped. Also, this is still "does" vs "do"?

Reply:
Still does.